### PR TITLE
csv-parsers

### PR DESCRIPTION
**Pull request by kghamilton89**
# 5486

### CODE REVIEW

File: parsers/csv/README.md [hunk 1]
```diff
@@ -2,60 +2,60 @@
 
 ## Overview
 
-CSV files can be [uploaded](uploading-csv-files.md) via HTTP API or manually through the web interface.
+[Upload CSV files](uploading-csv-files.md) via HTTP API or manually through the web interface.
 
 ![](./resources/csv.gif)
 
-To process a CSV file you need to create a CSV parser, which splits the file into lines (rows) consisting of multiple columns and converts the lines into `series`, `property`, and `message` commands.
+To process a CSV file, create a CSV Parser. The parser separates the file into rows and columns and then converts the information into `series`, `property`, and `message` commands.
```
Comment (rodionos) [inline parsers/csv/README.md]: Split is a formal operation the parser performs, so I would keep split instead of separate.

Comment (rodionos) [reply parsers/csv/README.md]: convert the information is too generic. Generally, information is too generic for our purposes. What parser does: a) split a multi-line file into lines which become rows. Then it slices (informal word) each line into tokens to make columns out of them. So the line becomes an array of cells. The values of these cells are mapped for our command fields and that's how commands are produced.

File: parsers/csv/README.md [hunk 2]
```diff
@@ -2,60 +2,60 @@
...
-|  `Timestamp Columns`  |  Columns containing the Timestamp for each data sample. In some cases, depending on the CSV file, the Timestamp can be split into multiple columns. For example: Date, Time.<br>If there are two columns containing the Timestamp, then they are concatenated with a dash symbol (-) in the Timestamp Pattern field.<br>For example:<br>Source CSV File:<br>`Date,Time,Sensor,Power`<br>`2015.05.15,09:15:00,sensor01,15`<br>Timestamp Columns:<br>`Date,Time`<br>Result:<br>`Date-Time`<br>`2015.05.15-09:15:00`<br>Timestamp Pattern Setting:<br>`yyyy.MM.dd-HH:mm:ss`  |
-|  `Timestamp Type`  |  Pattern, Seconds (Unix Seconds), Milliseconds (Unix Milliseconds).  |
-|  `Predefined Pattern`  |  Predefined Timestamp formats.  |
-|  `Timestamp Pattern`  |  Custom timestamp format, specified manually. For example: `dd-MMM-yy HH:mm:ss`<br>If there are two columns containing the Timestamp, then in they are divided with a dash (-) in the pattern.  |
+|  `Enabled`  | The database discards CSV files which reference a disabled parser.  |
+|  `Name`  |  Unique parser identifier.  |
+|  `Command Type`  |  Data contents: `Series`, `Message`, or `Property`.  |
```
Comment (rodionos) [inline parsers/csv/README.md]: Command type is the type of command produced after the file is split into lines and processed.

File: parsers/csv/README.md (hunk 1 above)
Comment (kghamilton89) [reply parsers/csv/README.md]: Changed.

**Comment by kghamilton89** [comment]
* PR contains only modifications to **Overview** section of CSV Parser.
* Removed my name from ATSD `README.md` title page image and reduced the size of technology stack image.

**Pull request merged**